### PR TITLE
Added user profile page

### PR DESCRIPTION
**Pull request by thbisdikian**
image

### CODE REVIEW

**Comment by devonbNU** [comment]
I think for consistency it would be best if it had some margin

File: src/css/profile-stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+.profile-image{
+	float:left;
+	height:100%;
+	width:25%;
+	overflow:hidden;
+}
+
+.profile-image img{
+	width:100%;
+	height:auto;
```
Comment (jholm117) [inline src/css/profile-stylesheet.css]: why is height auto and not 100%?

File: src/css/profile-stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	width: 100%;
+	height: 20rem;
+	margin-bottom:2rem;
+}
+
+.profile-image{
+	float:left;
+	height:100%;
+	width:25%;
+	overflow:hidden;
```
Comment (jholm117) [inline src/css/profile-stylesheet.css]: why do we need this

**Comment by thbisdikian** [comment]
image

**Pull request merged**